src/Sylius/Behat/Context/Ui/Admin/ManagingExchangeRatesContext.php

```diff
@@ -157,7 +159,7 @@ public function iDeleteThem(): void
     /**
      * @Then I should see :count exchange rates on the list
      */
-    public function iShouldSeeExchangeRatesOnTheList($count = 0)
+    public function iShouldSeeExchangeRatesOnTheList(int $count = 0): void
```

Comment:
This step does not allow a default value, so the method parameter could also reflect this.
```suggestion
    public function iShouldSeeExchangeRatesOnTheList(int $count): void
```